src/components/helpers/EnvironmentsList/EnvironmentsList.js

```diff
@@ -20,14 +20,22 @@ const EnvironmentsList = ({
       );
   return (
     <span>
-      {environments && environments.length > 0
-        ? environments.map(env =>
-            <EnvironmentsListItem
-              key={env.id}
-              runtimeEnvironment={env}
-              longNames={longNames}
-            />
-          )
+      {environments !== null
+        ? environments.length > 0
+          ? environments.map(env =>
+              <EnvironmentsListItem
+                key={env.id}
+                runtimeEnvironment={env}
+                longNames={longNames}
+              />
+            )
+          : <i className="small text-muted">
+              <WarningIcon gapRight />
+              <FormattedMessage
+                id="app.environmentsList.noEnvironments"
+                defaultMessage="no runtime environments"
```

Comment:
Capital leading `n`? But I'm not sure with the context...

Reply:
Well, the text is somewhat lead with the icon, so I decided to keep the caption lowercased. It can be changed, I do not have strong feelings about it.